dokomoforms/handlers/util.py

```diff
@@ -132,20 +132,25 @@ def _get_current_user_prefs(self):
             prefs = self.current_user_model.preferences
         return json_encode(prefs)
 
-    def _t(self, field):
+    def _t(self, field, default_language=None):
         """Pick a translation from a translatable field.
 
         Based on user's preference.
+
         Falls back to the first available translation if default_language
         is not available.
 
         TODO: this should probably fallback to the survey's default, if the
         string is coming from a survey...?
         """
+
         if self.current_user_model is not None:
             lang = self.current_user_model.preferences['default_language']
             if lang in field:
                 return field[lang]
+            elif default_language is not None:
+                if default_language in field:
+                    return field[default_language]
             return next(iter(field.values()))
```

Comment:
@jmwohl I recommend getting rid of the `next(iter(field.values()))` behavior since it doesn't do what it looks like it does. Instead, `default_language` should be a required argument.